src/components/error/ErrorBoundary.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Container from 'react-bootstrap/Container';
+
+class ErrorBoundary extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = { hasError: false };
+    }
+
+    static getDerivedStateFromError(error) {
+        return { hasError: true };
+    }
+
+    componentDidCatch(error, errorInfo) {
+        console.error(error, errorInfo)
+    }
+
+    render() {
+        if (this.state.hasError) {
+            return (
+                <Container>
+                    <h1>Что-то пошло не так.</h1>;
```

Comment:
Place ';' inside \</h1\> as it hangs on the next line :)
<img width="1360" alt="Screen Shot 2021-03-05 at 7 37 02 PM" src="https://user-images.githubusercontent.com/22471061/110145174-33371800-7dea-11eb-9258-217a3e17aed9.png">